src/OrchardCore/OrchardCore.Abstractions/Localization/LocalizationCookieName.cs

```diff
@@ -0,0 +1,19 @@
+using Microsoft.AspNetCore.Localization;
+
+namespace OrchardCore.Localization;
+
+/// <summary>
+/// Contains a localization cookie names.
+/// </summary>
+public class LocalizationCookieName
+{
+    /// <summary>
+    /// Gets the cookie name that used for the admin site.
+    /// </summary>
+    public static readonly string AdminSite = ".OrchardCore.AdminSiteCulture";
```

Comment:
For me `AdminSite` means nothing, maybe `AdminArea` or `AdminCulture` or here just `Admin`.

`.OrchardCore.AdminSiteCulture` => `.OrchardCore.AdminCulture`

    => /// Gets the cookie name that is used for the admin culture.

But as said in another comment the cookie name needs to be prefixed or suffixed per tenant, e.g. by using the shell settings `VersionId`.

Reply:
> For me AdminSite means nothing, maybe AdminArea or AdminCulture or here just Admin.

I remembered we are using TheSite & TheAdminSite a lot

> But as said in another comment the cookie name needs to be prefixed or suffixed per tenant, e.g. by using the shell settings VersionId.

Why? the path can differentiate each tenant if I'm not wrong

Reply:
> I remembered we are using TheSite & TheAdminSite a lot

No, we are using e.g. `SiteSettings` and `AdmiSettings` but not the above terms.

> Why? the path can differentiate each tenant if I'm not wrong

Not sure, maybe the path is just to select the more specific ones, would need to be tried, for me better to differentiate by the cookie name.

But as said in a previous comment not so good to use the shell name for the cookie path, refer to https://github.com/OrchardCMS/OrchardCore/pull/12017#discussion_r931541558

Reply:
> No, we are using e.g. SiteSettings and AdmiSettings but not the above terms.

Many times I heard Seb mentioned the terms in the OC standup ;)

> But as said in a previous comment not so good to use the shell name for the cookie path, refer to

I will consider, if we do so we need to check other places where the settings name has been used

Reply:
Here I was talking about the cookie path, we never used the setting name for a cookie **path**.